lib/grunt/task.js

```diff
@@ -1,5 +1,11 @@
 'use strict';
 
+// Keep track of the number of log.error() calls.
+var errorcount,
+
+  // The last specified tasks message.
+  lastInfo;
```

Comment:
This is a difference between JSCS and ESLint `one-var`/`disallowMultipleVarDecl` rules, i figure it is easier to change the code then to propose changes in ESLint